### ISSUE TITLE
All Replays from DoTa Cannot Parse since 3/6 update

### ISSUE THREAD

**Issue by STRATZ-Ken** [skadistats/clarity#294]
```
skadistats.clarity.ClarityException: Entity not found for update at index 154. Entity update cannot be parsed!
    at skadistats.clarity.processor.entities.Entities.processPacketEntities(Entities.java:409)
    at skadistats.clarity.processor.entities.Entities.processAndRunPacketEntities(Entities.java:337)
    at skadistats.clarity.processor.entities.Entities.onPacketEntities(Entities.java:332)
    at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86)
    at skadistats.clarity.event.Event.raise(Event.java:41)
    at skadistats.clarity.processor.reader.InputSourceProcessor.processEmbedded(InputSourceProcessor.java:247)
    at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86)
    at skadistats.clarity.event.Event.raise(Event.java:41)
    at skadistats.clarity.processor.reader.InputSourceProcessor.processSource(InputSourceProcessor.java:174)
    at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86)
    at skadistats.clarity.event.Event.raise(Event.java:41)
    at skadistats.clarity.processor.runner.AbstractFileRunner.initAndRunWith(AbstractFileRunner.java:39)
    at skadistats.clarity.processor.runner.SimpleRunner.runWith(SimpleRunner.java:33)
    at Worker$1.handleDelivery(Worker.java:149)
    at com.rabbitmq.client.impl.ConsumerDispatcher$5.run(ConsumerDispatcher.java:149)
    at com.rabbitmq.client.impl.ConsumerWorkService$WorkPoolRunnable.run(ConsumerWorkService.java:111)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.lang.Thread.run(Thread.java:833)
```

**Comment by spheenik** [comment]
Fix is in pipeline.

**Comment by spheenik** [comment]
Released version 2.7.7 with the fix.
